backend/src/seed/factories/donations.js

```diff
@@ -0,0 +1,18 @@
+import uuid from 'uuid/v4'
+
+export default function create() {
+  return {
+    factory: async ({ args, neodeInstance }) => {
+      const defaults = {
+        id: uuid(),
+        goal: 15000,
+        progress: 0,
+      }
+      args = {
+        ...defaults,
+        ...args,
+      }
+      return neodeInstance.create('Donations', args)
```

Comment:
Usually I would prefer singular here. Like factory bot: https://github.com/thoughtbot/factory_bot The file is plural but the model is singular.

I think this model is an exception and I'm OK with it here.

Reply:
yeah, I thought of that as well, struggled with it, and just decided it could be `Donations` since it's not a single donation, but an aggregation of donations.